src/index.ts

```diff
@@ -37,42 +34,34 @@ const init: InitCMP = ({
 		return;
 	}
 
-	if (typeof isInUsa !== 'undefined') {
-		country = isInUsa ? 'US' : 'GB';
-
-		console.warn(
-			'`isInUsa` will soon be deprecated. Prefer using `country` instead.',
-		);
-	}
-
-	if (typeof country === 'undefined') {
-		throw new Error(
-			'CMP initialised without `country` property. A 2-letter, ISO ISO_3166-1 country code is required.',
-		);
+	const locale = await getLocale();
+
+	if (locale) {
+		const framework = getFramework(locale);
+
+		window.guCmpHotFix.initialised = true;
+
+		switch (framework) {
+			case 'ccpa':
+				CMP = CCPA;
+				break;
+			case 'aus':
+				CMP = AUS;
+				break;
+			case 'tcfv2':
+			default:
+				// default is also 'tcfv2'
+				CMP = TCFv2;
+				break;
+		}
+
+		setCurrentFramework(framework);
+
+		CMP.init(pubData ?? {});
```

Comment:
Is it even worth passing an empty object?

```suggestion
		CMP.init(pubData);
```